src/routes/Project/actions/ClipActionHandlers.js

```diff
@@ -0,0 +1,13 @@
+import ProjectConstants from '../constants';
+
+const ACTION_HANDLERS = {
+  [ProjectConstants.ADD_CLIP]: (state, action) => {
```

Comment:
we're using babel right? I do like this syntax but it's definitely not widely supported.